monitoring/ceph-mixin/dashboards/rgw.libsonnet

```diff
@@ -1,825 +1,879 @@
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
       )
-      .addRequired(
-        type='panel',
-        id='grafana-piechart-panel',
-        name='Pie Chart',
-        version='1.3.3'
+      .addTargets(
+        [
+          $.addTargetSchema(
+            |||
+              sum(
+                rate(
+                  haproxy_frontend_bytes_out_total{proxy=~"frontend", job=~"$job_haproxy", instance=~"$ingress_service"}[$__rate_interval]
+                ) * 8
+              ) by (instance)
+            |||, 'OUT Front', 'time_series', 2
+          ),
+          $.addTargetSchema(
+            |||
+              sum(
+                rate(
+                  haproxy_backend_bytes_in_total{proxy=~"backend", job=~"$job_haproxy", instance=~"$ingress_service"}[$__rate_interval]
+                ) * 8
+              ) by (instance)
+            |||, 'IN Back', 'time_series', 2
+          ),
+          $.addTargetSchema(
+            |||
+              sum(
+                rate(
+                  haproxy_backend_bytes_out_total{proxy=~"backend", job=~"$job_haproxy", instance=~"$ingress_service"}[$__rate_interval]
+                ) * 8
+              ) by (instance)
+            |||, 'OUT Back', 'time_series', 2
+          ),
+        ]
       )
-      .addRequired(
-        type='panel', id='graph', name='Graph', version='5.0.0'
+      .addSeriesOverride([
+        {
+          alias: '/.*OUT.*/',
+          transform: 'negative-Y',
+        },
+      ]),
+    ]),
+  'radosgw-detail.json':
+    local RgwDetailsPanel(aliasColors,
+                          title,
+                          description,
+                          formatY1,
+                          formatY2,
+                          expr1,
+                          expr2,
+                          legendFormat1,
+                          legendFormat2,
+                          x,
+                          y,
+                          w,
+                          h) =
+      $.graphPanelSchema(aliasColors,
+                         title,
+                         description,
+                         'null',
+                         false,
+                         formatY1,
+                         formatY2,
+                         null,
+                         null,
+                         0,
+                         1,
+                         '$datasource')
+      .addTargets(
+        [$.addTargetSchema(expr1, legendFormat1), $.addTargetSchema(expr2, legendFormat2)]
+      ) + { gridPos: { x: x, y: y, w: w, h: h } };
+
+    $.dashboardSchema(
+      'RGW Instance Detail',
+      '',
+      'x5ARzZtmk',
+      'now-1h',
+      '15s',
+      16,
+      $._config.dashboardTags + ['overview'],
+      '',
+      {
+        refresh_intervals: ['5s', '10s', '15s', '30s', '1m', '5m', '15m', '30m', '1h', '2h', '1d'],
+        time_options: ['5m', '15m', '1h', '6h', '12h', '24h', '2d', '7d', '30d'],
+      }
```

Comment:
This is repeated a lot. Any way to reuse these?

Reply:
That's almost the default, I removed all of these